openapi.go

```diff
@@ -79,6 +79,28 @@ func NewOpenApiSpec() openapi3.T {
 	return spec
 }
 
+type OpenAPIServable interface {
+	SpecHandler()
+	UIHandler()
+	URL() string
+}
+
+func RegisterOpenAPIRoutes(e *Engine, o OpenAPIServable) {
```

Comment:
Could be just a method

Reply:
Also the user can either just call this or do we want to eventially wrap our adaptors with a `Run` that calls this for them.

Reply:
I imagine this will eventually just be called from an `Adaptor`s run command in the future.